gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Oct 12 16:55:20 SGT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
```

Comment:
Not sure why this is changed though. The local + org's on my side is the bin version